### DISCUSSION TITLE
Hindi transcription question

### DISCUSSION THREAD

**Discussion by cjaff** [deepgram/community#706] [General help] [answered]
For Hindi transcription, is the output in Devanagari script or romanized script?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @cjaff, you can receive Devanagari output with `language=hi&model=nova-2` (`hi` language code), or Romanized/Latin script with `language=hi-Latn&model=nova-2` (`hi-Latn` language code).

**Comment by cjaff** [comment]
Thank you for the help!

On Tue, Apr 16, 2024 at 5:10 PM Julia Kroll ***@***.***>
wrote:

> Hi @cjaff <https://github.com/cjaff>, you can receive Devanagari output
> with language=hi&model=nova-2 (hi language code), or Romanized/Latin
> script with language=hi-Latn&model=nova-2 (hi-Latn language code).
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/deepgram/discussions/706#discussioncomment-9135241>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BH34WKPPBFHBZAZPYYZT2N3Y5WHV3AVCNFSM6AAAAABGJXUOUGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TCMZVGI2DC>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

[image: -] *Cristina Jaffery*
Product Manager
Discuss.io
<https://www.discuss.io/?utm_source=signaturesatori&utm_medium=email&utm_campaign=general_signature>
*E * ***@***.*** | *M * 8455534232
